cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaConnectApi.java

```diff
@@ -469,4 +484,135 @@ void rescheduleOrComplete(Long tid) {
                 .end();
         return result.future();
     }
+
+    private Future<Void> updateConnectorLogger(String host, int port, String logger, String level) {
+        Promise<Void> result = Promise.promise();
+        HttpClientOptions options = new HttpClientOptions().setLogActivity(true);
+        String path = "/admin/loggers/" + logger;
+        JsonObject levelJO = new JsonObject();
+        levelJO.put("level", level);
+        log.debug("Making PUT request to {} with body {}", path, levelJO);
+        vertx.createHttpClient(options)
+                .put(port, host, path, response -> {
+                    response.exceptionHandler(error -> {
+                        result.fail(error);
+                    });
+                    response.bodyHandler(body -> {
+                    });
+                    if (response.statusCode() == 200) {
+                        log.debug("Logger {} updated to level {}", logger, level);
+                        result.complete();
+                    } else {
+                        log.debug("Logger {} did not update to level {} (http code {})", logger, level, response.statusCode());
+                        result.fail(new ConnectRestException(response, "Unexpected status code"));
+                    }
+                })
+                .exceptionHandler(result::fail)
+                .putHeader("Content-Type", "application/json")
+                .putHeader("Content-Length", String.valueOf(levelJO.toBuffer().length()))
+                .setFollowRedirects(true)
+                .write(levelJO.toBuffer())
+                .end();
+
+        return result.future();
+    }
+
+    @SuppressWarnings("unchecked")
```

Comment:
What exactly are we suppressing? Any way to avoid it? Maybe we could start adding comments to explain.